### PR TITLE
add padding for pages without it

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

File: web/public/css/dashboard.less [hunk 1]
```diff
@@ -660,10 +666,6 @@ label {
 
 .cd-dojo-main {
 
-  .cd-input-row {
-    margin: 20px 0 35px;
```
Comment (Wardormeur) [inline web/public/css/dashboard.less]: does this affect me or not then? :>

File: web/public/css/dashboard.less [hunk 2]
```diff
@@ -1820,3 +1817,24 @@ span.applications-banner {
     13em 22em,
     14em 22em;
 }
+// Classes made to fix padding should eventually be moved for coresponding dierectives
+.my-dojos {
+  padding: 0 32px;
```
Comment (Wardormeur) [inline web/public/css/dashboard.less]: use a mixin to store your common padding :)

Comment (Wardormeur) [inline web/public/css/dashboard.less]: also, isn't 32px too much for responsiveness?

File: web/public/css/dashboard.less (hunk 1 above)
Comment (butlerx) [inline web/public/css/dashboard.less]: nope it shouldnt
found this in 2 places

File: web/public/css/dashboard.less (hunk 2 above)
Comment (butlerx) [inline web/public/css/dashboard.less]: nope works well for responsive

File: web/public/css/dashboard.less [hunk 3]
```diff
@@ -1820,3 +1817,27 @@ span.applications-banner {
     13em 22em,
     14em 22em;
 }
+// Classes made to fix padding should eventually be moved for coresponding dierectives
+.padding {
```
Comment (DanielBrierton) [inline web/public/css/dashboard.less]: Should have a more descriptive name. You couldn't use this as padding for a button for example.

**Pull request merged**